kod/util/factgame/territry.kod

```diff
@@ -175,8 +175,6 @@ messages:
          else
          {
             send(self,@CountFlags);
-            % do this after, so that bonus town recalcs saved
-            send(self,@SaveFlagStates);
```

Comment:
Just double-checking: previously, SaveFlagStates wasn't called for the "partial" game mode, but now it is.  That's OK?

Reply:
🤔 I need to revisit `Recreate` because this doesn't address recreating the partial game (which was why I added this call to save here). 🤦